webviz_subsurface/_providers/ensemble_table_provider_factory.py

```diff
@@ -0,0 +1,337 @@
+from typing import Dict, Optional
+from pathlib import Path
+import os
+import hashlib
+import json
+import logging
+import pickle  # nosec
+from enum import Enum
+
+import pandas as pd
+from fmu.ensemble import ScratchEnsemble
+
+from webviz_config.webviz_factory_registry import WEBVIZ_FACTORY_REGISTRY
+from webviz_config.webviz_factory import WebvizFactory
+from webviz_config.webviz_instance_info import WebvizRunMode
+
+from .ensemble_table_provider import EnsembleTableProvider
+from .ensemble_table_provider import EnsembleTableProviderSet
+from .ensemble_table_provider_impl_arrow import EnsembleTableProviderImplArrow
+from .ensemble_table_provider_impl_inmem_parquet import (
+    EnsembleTableProviderImplInMemParquet,
+)
+from .._utils.perf_timer import PerfTimer
+
+
+class BackingType(Enum):
+    ARROW = "arrow"
+    INMEM_PARQUET = "inmem_parquet"
+
+
+LOGGER = logging.getLogger(__name__)
+
+
+def _make_hash_string(string_to_hash: str) -> str:
+    # There is no security risk here and chances of collision should be very slim
+    return hashlib.md5(string_to_hash.encode()).hexdigest()  # nosec
+
+
+# =============================================================================
```

Comment:
Remove these type of comments since we don't have them elsewhere in the code style?

Reply:
Fixed in db7fa7d and 47a7106